hedera-node/hedera-app/src/main/java/com/hedera/node/app/Hedera.java

```diff
@@ -687,22 +670,6 @@ private void doGenesisHousekeeping() {
          */
     }
 
-    private void buildStakingInfoMap(
```

Comment:
Has this functionality moved elsewhere? Or how do we initialize staking info?

Reply:
Oh, it is done during schema migration. 🤦‍♂️

Reply:
Yes, it is handled by the genesis schema in the token service, as per @mhess-swl